### ISSUE TITLE
`CKV_AZURE_4` cannot work with AzureRM 3.0 provider

### ISSUE THREAD

**Issue by lonegunmanb** [bridgecrewio/checkov#3159]
**Describe the issue**

Scan an aks resource with AzrueRM 3.0 provider will get the following false error:

```text
terraform scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 0

Check: CKV_AZURE_4: "Ensure AKS logging to Azure Monitoring is Configured"
        FAILED for resource: azurerm_kubernetes_cluster.example
        File: /main.tf:35-58
        Guide: https://docs.bridgecrew.io/docs/bc_azr_kubernetes_1

                35 | resource "azurerm_kubernetes_cluster" "example" {
                36 |   name                = "example-aks1"
                37 |   location            = azurerm_resource_group.example.location
                38 |   resource_group_name = azurerm_resource_group.example.name
                39 |   dns_prefix          = "exampleaks1"
                40 |
                41 |   default_node_pool {
                42 |     name       = "default"
                43 |     node_count = 1
                44 |     vm_size    = "Standard_D2_v2"
                45 |   }
                46 |
                47 |   oms_agent {
                48 |     log_analytics_workspace_id = azurerm_log_analytics_workspace.main.id
                49 |   }
                50 |
                51 |   identity {
                52 |     type = "SystemAssigned"
                53 |   }
                54 |
                55 |   tags = {
                56 |     Environment = "Production"
                57 |   }
                58 | }
```

**Examples**
```hcl
variable "admin_user" {
  default = null
}

provider "azurerm" {
  features {}
}

resource "azurerm_resource_group" "example" {
  location = "eastus"
  name     = "checkov-dynamic"
}

resource "azurerm_log_analytics_workspace" "main" {
  name                = "workspace"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  sku                 = "PerGB2018"
  retention_in_days   = 7
}

resource "azurerm_log_analytics_solution" "main" {
  solution_name         = "ContainerInsights"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  workspace_resource_id = azurerm_log_analytics_workspace.main[0].id
  workspace_name        = azurerm_log_analytics_workspace.main[0].name

  plan {
    publisher = "Microsoft"
    product   = "OMSGallery/ContainerInsights"
  }
}

resource "azurerm_kubernetes_cluster" "example" {
  name                = "example-aks1"
  location            = azurerm_resource_group.example.location
  resource_group_name = azurerm_resource_group.example.name
  dns_prefix          = "exampleaks1"

  default_node_pool {
    name       = "default"
    node_count = 1
    vm_size    = "Standard_D2_v2"
  }

  oms_agent {
    log_analytics_workspace_id = azurerm_log_analytics_workspace.main.id
  }

  identity {
    type = "SystemAssigned"
  }

  tags = {
    Environment = "Production"
  }
}
```

**Version (please complete the following information):**
 - 2.1.20

**Additional context**

The [current implemention](https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/azure/AKSLoggingEnabled.py#L13-L14) are based on 2.x and not work for 3.x.
